src/gui/plugins/global_illumination_civct/GlobalIlluminationCiVct.qml

```diff
@@ -0,0 +1,221 @@
+      }
+    }
+
+  Button {
+    id: resetCascades
+    text: qsTr("Reset Cascades")
+    enabled: !GlobalIlluminationCiVct.cascadesEditable
+    Layout.columnSpan: 6
+    Layout.fillWidth: true
+    onClicked: {
+      GlobalIlluminationCiVct.ResetCascades()
+    }
+  }
+
+  CheckBox {
+    Layout.alignment: Qt.AlignHCenter
+    id: displayVisual
+    Layout.columnSpan: 6
+    Layout.fillWidth: true
+    text: qsTr("Enabled")
+    checked: GlobalIlluminationCiVct.enabled
+    onToggled: {
+      GlobalIlluminationCiVct.enabled = checked
+    }
+  }
+
+  Text {
+    Layout.columnSpan: 4
+    id: bounceCountStr
+    color: "dimgrey"
+    text: qsTr("BounceCount")
+  }
+
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: bounceCount
+    value: GlobalIlluminationCiVct.bounceCount
+    minimumValue: 0
+    maximumValue: 16
+    decimals: 1
+    onValueChanged: {
+      GlobalIlluminationCiVct.bounceCount = value
+    }
+  }
+
+  CheckBox {
+    Layout.alignment: Qt.AlignHCenter
+    id: highQuality
+    Layout.columnSpan: 6
+    Layout.fillWidth: true
+    text: qsTr("High Quality")
+    checked: GlobalIlluminationCiVct.highQuality
+    onToggled: {
+      GlobalIlluminationCiVct.highQuality = checked;
+    }
+  }
+
+  CheckBox {
+    Layout.alignment: Qt.AlignHCenter
+    id: anisotropic
+    Layout.columnSpan: 6
+    Layout.fillWidth: true
+    text: qsTr("Anisotropic")
+    checked: GlobalIlluminationCiVct.anisotropic
+    onToggled: {
+      GlobalIlluminationCiVct.anisotropic = checked;
+    }
+  }
+
+  Text {
+    Layout.columnSpan: 2
+    id: debugVisualizationStr
+    color: "dimgrey"
+    text: qsTr("DebugVisualization")
+  }
+
+  ComboBox {
+    Layout.columnSpan: 4
+    id: debugVisualization
+    Layout.fillWidth: true
+    currentIndex: GlobalIlluminationCiVct.debugVisualizationMode
+    model: ["Albedo", "Normal", "Emissive", "Lighting", "None"]
+    onCurrentIndexChanged: {
+      if (currentIndex < 0|| currentIndex > 4) {
+        return;
+      }
+      GlobalIlluminationCiVct.debugVisualizationMode = currentIndex
+    }
+  }
+
+  RoundButton {
+    Layout.columnSpan: 1
+    text: "\u21bb"
+    Material.background: Material.primary
+    onClicked: {
+      combo.currentIndex = 0
+      GlobalIlluminationCiVct.OnRefreshCameras();
+    }
+    ToolTip.visible: hovered
+    ToolTip.delay: tooltipDelay
+    ToolTip.timeout: tooltipTimeout
+    ToolTip.text: qsTr("Refresh list all available cameras")
+  }
+
+  ComboBox {
+    Layout.columnSpan: 5
+    id: combo
+    Layout.fillWidth: true
+    model: GlobalIlluminationCiVct.cameraList
+    currentIndex: 0
+    onCurrentIndexChanged: {
+      if (currentIndex < 0)
+        return;
+      GlobalIlluminationCiVct.OnCamareBind(textAt(currentIndex));
+    }
+    ToolTip.visible: hovered
+    ToolTip.delay: tooltipDelay
+    ToolTip.timeout: tooltipTimeout
+    ToolTip.text: qsTr("Camera around which all cascades are centered from")
+  }
+}
+
+/*##^##
```

Comment:
remove

Reply:
Same here, QtDesigner keeps adding this back. Should we keep it?